docs/software/spack.md

```diff
@@ -0,0 +1,157 @@
+
+You can also modify the installation commands to meet your installation needs. You can specify which version you'd like installed using the `@` operator:
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack install --add fastqc@0.11.9
+```
+
+#### Installation Locations and Specifics
+
+The root of the Spack installation tree is `/projects/$USER/software/spack`. From there, executables can be found in subdirectories depending on the OS and compiler used to install the software. Software will be installed here regardless of whether the software was installed in an environment. 
+
+Spack will always check the installation root prior to installing software, in an environment or otherwise. If the software is not found in the installation root, it will be installed there. If the software is found in the installation root, it will not be installed. When attempting to install in an environment, symbolic links to the installation root will be created in `/projects/$USER/spack/environments/<envname>/.spack-env/view`. This behavior means that once a piece of software is installed, it can be linked to any environment without having to perform the installation again. 
+
+The default cache is located at `/scratch/alpine/$USER/spack/cache`. This directory will contain any previously-installed tarballs and repositories, as well as any miscellaneous cache items.  
+
+### Installing and Using Compilers with Spack
+
+In addition to standard software packages, you can use Spack to install compilers which are not currently available on the CURC software stack. For example, if you need the newest available version of `gcc`, you can install it into your environment as follows: 
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack install --add gcc@13.1.0
+```
+
+Once the installation is complete, you can make it available to Spack with `spack compiler add`:
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack compiler add <path to compiler>
+```
+
+>Note that, in this case, you can locate the path of the compiler with `spack location -i gcc@13.1.0`.
+
+Once the compiler is added, you can install any subsequent packages using the compiler you've installed with the `%` operator. For example:
+
+```
+[johndoe@c3cpu-c11-u17-2 ~]$ spack install --add fastqc%gcc@13.1.0
+```
+
+### Key Spack Terms: 
+
+| Term | Definition |
+|---------|----------|
+| Spec | Description of the build requirements for an installation. This includes the software, version, compiler, and other variants. |
+| Variant | An option or feature that can be toggled for a specific package. These usually correspond to a configuration or compilation setting. |
+| Repo | A central location which stores packages. Repo locations can be configured. |
+| Environment | A collection of packages and dependencies that are available to a user. In Spack, environments do not interact. |
+
+### Basic Spack commands:
+
+| Command | Function |
+|---------|----------|
```

Comment:
add:
spack env list (list all your environments)
spack remove <env> (remove an environment) 
spack uninstall <packagename> remove a package - though this can get complicated so maybe link these uninstall docs - https://spack-tutorial.readthedocs.io/en/latest/tutorial_environments.html#uninstalling-packages
spack env status <check which env you're in>